financial-connections/src/main/java/com/stripe/android/financialconnections/features/consent/ConsentViewModel.kt

```diff
@@ -128,7 +128,7 @@ internal class ConsentViewModel @AssistedInject constructor(
             return defaultDestination
         }
 
-        val hasExistingAccount = hasExistingLinkAccount(manifest, prefillDetails.email)
+        val hasExistingAccount = hasExistingLinkAccount(manifest, prefillDetails.email!!)
```

Comment:
Oh, is non-nullability not deferred from the `useManifestNextPane` computation? Is there something else we can do to avoid the `!!`?

Reply:
this is a side effect of the module updates:

```
Smart cast to 'kotlin. String' is impossible, because 'email' is a public API property declared in different module.
```

Reply:
Yeah, either that or allow a null email in `hasExistingLinkAccount`.